airbyte-integrations/connectors/source-freshcaller/acceptance-test-config.yml

```diff
@@ -2,6 +2,8 @@ connector_image: airbyte/source-freshcaller:dev
 tests:
   spec:
     - spec_path: "source_freshcaller/spec.json"
+      backward_compatibility_tests_config:
+        disable_for_version: 0.1.0 # Fix start_date type
```

Comment:
Were the backward compatibility tests failing before this change? I'm not sure the change you made are actually backward incompatible? What was the default type stored for date before you specifically declare it's a string?

Reply:
`Start_date` field was not declared as a string in jsonschema, but was rendered as a string in UI.
This possible caused a pattern validation error (see screenshot from https://github.com/airbytehq/airbyte/issues/25688) 
error without disabling:
![image](https://github.com/airbytehq/airbyte/assets/36314070/756faeb8-6ddd-4c24-b605-8d9560996d2d)